activerecord/lib/active_record/connection_adapters/schema_cache.rb

```diff
@@ -141,7 +141,7 @@ def clear_data_source_cache!(name)
       def dump_to(filename)
         clear!
         connection.data_sources.each { |table| add(table) }
-        open(filename, "wb") { |f|
+        open(filename) { |f|
```

Comment:
Could [`File.atomic_write`](https://github.com/rails/rails/blob/69d25a5eba67b2556ffe01c515abd2a54ad88003/activesupport/lib/active_support/core_ext/file/atomic.rb) work as well here?

Reply:
Nice, I wasn't aware of `atomic_write`. Thanks, yeah I think we might as well use that!